Worktrolly/worktrolly
# Dev/validation

### Functionality:
Validation as a service

### Solution:
Calling the validation service on submitting form and displaying any missing field in console.

### Risk level:

### How to test:
1. ng serve
2. On skipping any field in the create-new-session, create-new-sprint, edit-page and log-work component, a message stating the missing field is displayed in the console.

The missing field is displayed:
![Screenshot (11)](https://user-images.githubusercontent.com/52309262/104820804-671ba580-585d-11eb-95ce-eebc772fccdb.png)
The task is created only after all fields are entered
![Screenshot (13)](https://user-images.githubusercontent.com/52309262/104820822-861a3780-585d-11eb-9e4f-34e13efeb33f.png)
